graylog2-web-interface/package.json

```diff
@@ -90,9 +78,8 @@
     "less-loader": "^2.2.1",
     "mocha": "^2.3.4",
     "phantomjs-prebuilt": ">=1.9",
-    "react-hot-loader": "^1.3.0",
+    "react-hot-loader": "^3.0.0-beta.3",
```

Comment:
What are the reasons to use a beta version here? I guess you already thought about it and have good reasons, but I was wondering why we want to upgrade to version 3 beta.